core/src/main/java/bio/terra/pearl/core/service/site/SiteImageService.java

```diff
@@ -40,6 +40,7 @@ public List<SiteImageMetadata> findMetadataByPortal(String portalShortcode) {
 
     @Override
     public SiteImage create(SiteImage image) {
+
```

Comment:
It feels like now when clean name is now provided we can remove the part where we reprocess the clean file name?

Reply:
there are other places that call this function (e.g. the populate code), so I'd just as soon keep the reprocessing in for safety.